azure-pipelines/e2e-assets/asset-caching/success-script.ps1

```diff
@@ -0,0 +1,22 @@
+param (
+    [string]$Url,
+    [string]$ExpectedSHA512,
+    [string]$Destination
+)
+
+# Ensure the destination folder exists
+$downloadFolder = Split-Path -Path $Destination -Parent
+if (-not (Test-Path $downloadFolder)) {
+    New-Item -Path $downloadFolder -ItemType Directory | Out-Null
+}
+
+# Download the file directly to the specified destination path
+Invoke-WebRequest -Uri $Url -OutFile $Destination
```

Comment:
Can you fix this test to avoid needing to hit the network? The existing `$TestScriptAssetCacheExe` does not need that.

Reply:
I'll take a look. This was just the simplest test I can think of